[2023-10-30T11:22:42.590+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_names.streaming_to_kafka manual__2023-10-30T11:22:38.401776+00:00 [queued]>
[2023-10-30T11:22:42.598+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_names.streaming_to_kafka manual__2023-10-30T11:22:38.401776+00:00 [queued]>
[2023-10-30T11:22:42.598+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-30T11:22:42.636+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): streaming_to_kafka> on 2023-10-30 11:22:38.401776+00:00
[2023-10-30T11:22:42.648+0000] {standard_task_runner.py:57} INFO - Started process 301 to run task
[2023-10-30T11:22:42.657+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'random_names', 'streaming_to_kafka', 'manual__2023-10-30T11:22:38.401776+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpeap6bzzj']
[2023-10-30T11:22:42.662+0000] {standard_task_runner.py:85} INFO - Job 47: Subtask streaming_to_kafka
[2023-10-30T11:22:42.762+0000] {task_command.py:415} INFO - Running <TaskInstance: random_names.streaming_to_kafka manual__2023-10-30T11:22:38.401776+00:00 [running]> on host df75f57c0534
[2023-10-30T11:22:43.107+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='random_names' AIRFLOW_CTX_TASK_ID='streaming_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2023-10-30T11:22:38.401776+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-30T11:22:38.401776+00:00'
[2023-10-30T11:22:44.428+0000] {logging_mixin.py:151} INFO - {'full name': 'Mr Jimmie Gilbert', 'gender': 'male', 'location': 'Eason Rd 3471', 'city': 'Warragul', 'country': 'Australia', 'postcode': 1545, 'latitude': '-86.8994', 'longtitude': '-40.1430', 'email': 'jimmie.gilbert@example.com'}
[2023-10-30T11:22:44.436+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.22.0.5', 29092)]>: connecting to kafka:29092 [('172.22.0.5', 29092) IPv4]
[2023-10-30T11:22:44.437+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2023-10-30T11:22:44.439+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.22.0.5', 29092)]>: Connection complete.
[2023-10-30T11:22:44.687+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2023-10-30T11:22:44.688+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2023-10-30T11:22:44.696+0000] {logging_mixin.py:151} INFO - KafkaProducer connected successfully.
[2023-10-30T11:22:44.852+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:44.853+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:44.853+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:44.854+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:44.956+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:44.957+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:44.958+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:44.959+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:45.061+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:45.062+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:45.063+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:45.063+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:45.165+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:45.165+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:45.165+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:45.166+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:45.318+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:45.352+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:45.352+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:45.353+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:45.506+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:45.507+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:45.508+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:45.509+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:45.814+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:45.814+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:45.814+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:45.815+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:46.118+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:46.119+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:46.119+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:46.120+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:46.524+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:46.525+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:46.526+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:46.526+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:46.931+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:46.931+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:46.931+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:46.931+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:47.688+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:47.690+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:47.690+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:47.690+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:48.448+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:48.449+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:48.450+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:48.450+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:49.663+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:49.665+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:49.666+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:49.667+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:50.878+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:50.879+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:50.879+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:50.879+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:52.042+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:52.045+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:52.046+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:52.047+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:53.208+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:53.210+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:53.210+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:53.211+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:54.220+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:54.221+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:54.221+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:54.221+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:55.230+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:55.230+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:55.231+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:55.231+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:56.493+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:56.494+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:56.494+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:56.495+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:57.755+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:22:57.756+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:22:57.756+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:22:57.757+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:22:58.966+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:22:58.967+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:22:58.967+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:22:58.967+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:00.176+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:00.176+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:00.177+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:00.177+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:01.340+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:01.341+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:01.342+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:01.343+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:02.505+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:02.505+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:02.506+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:02.507+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:03.518+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:03.520+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:03.520+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:03.521+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:04.532+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:04.532+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:04.532+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:04.532+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:05.743+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:05.745+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:05.746+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:05.747+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:06.960+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:06.961+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:06.962+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:06.962+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:07.823+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:07.824+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:07.825+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:07.825+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:08.684+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:08.685+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:08.685+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:08.685+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:09.594+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:09.595+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:09.595+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:09.596+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:10.505+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:10.506+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:10.507+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:10.509+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:11.570+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:11.571+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:11.571+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:11.572+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:12.633+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:12.634+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:12.635+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:12.635+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:13.847+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:13.848+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:13.848+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:13.849+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:15.061+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:15.062+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:15.062+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:15.063+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:16.275+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:16.277+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:16.277+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:16.278+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:17.492+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:17.492+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:17.493+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:17.494+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:18.606+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:18.607+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:18.607+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:18.608+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:19.719+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:19.720+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:19.720+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:19.721+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:20.682+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:20.683+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:20.684+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:20.684+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:21.644+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:21.644+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:21.645+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:21.645+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:22.756+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:22.757+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:22.757+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:22.757+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:23.819+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:23.820+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:23.820+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:23.821+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:24.679+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:24.680+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:24.681+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:24.682+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:25.541+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:25.542+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:25.543+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:25.543+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:26.605+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:26.606+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:26.606+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:26.606+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:27.666+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:27.667+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:27.668+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:27.669+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:28.781+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:28.782+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:28.782+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:28.782+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:29.894+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:29.894+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:29.895+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:29.896+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:31.056+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:31.057+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:31.057+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:31.057+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:32.215+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:32.215+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:32.215+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:32.216+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:33.324+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:33.324+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:33.324+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:33.325+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:34.433+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:34.433+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:34.433+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:34.434+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:35.694+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:35.695+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:35.696+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:35.696+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:36.957+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:36.957+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:36.958+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:36.959+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:38.221+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:38.222+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:38.222+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:38.223+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:39.485+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:39.485+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:39.485+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:39.486+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:40.547+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:40.547+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:40.548+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:40.548+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:41.608+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:41.609+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:41.609+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:41.610+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:42.772+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:23:42.773+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:23:42.773+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:23:42.774+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:43.937+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:23:43.938+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:23:43.938+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:23:43.939+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:23:44.697+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scripts/script.py", line 52, in start_streaming
    future = producer.send('random_names', value=data)
  File "/home/airflow/.local/lib/python3.8/site-packages/kafka/producer/kafka.py", line 576, in send
    self._wait_on_metadata(topic, self.config['max_block_ms'] / 1000.0)
  File "/home/airflow/.local/lib/python3.8/site-packages/kafka/producer/kafka.py", line 702, in _wait_on_metadata
    raise Errors.KafkaTimeoutError(
kafka.errors.KafkaTimeoutError: KafkaTimeoutError: Failed to update metadata after 60.0 secs.
[2023-10-30T11:23:44.804+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=random_names, task_id=streaming_to_kafka, execution_date=20231030T112238, start_date=20231030T112242, end_date=20231030T112344
[2023-10-30T11:23:44.866+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 47 for task streaming_to_kafka (KafkaTimeoutError: Failed to update metadata after 60.0 secs.; 301)
[2023-10-30T11:23:44.910+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-30T11:23:44.938+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
